src/lib/config/arbitrum/pools.ts

```diff
@@ -134,6 +134,8 @@ const pools: Pools = {
       '0xce2da1d3e5b5e4e1913f9ff65ee029d38682d8b900020000000000000000044e',
       '0x8bc65eed474d1a00555825c91feab6a8255c2107000000000000000000000453',
       '0x567ecfcb22205d279bb8eed3e066989902bf03d5000000000000000000000452',
+      '0xc7fa3a3527435720f0e2a4c1378335324dd4f9b3000200000000000000000459',
+      '0x01990f1e6f7f32296f125ee9469705c1c070054d000000000000000000000461',
```

Comment:
This pool hasn't had its init join done on it, and token image is missing for rETH.

Reply:
I've created a PR for rETH image here: https://github.com/balancer/tokenlists/pull/247/files